### DISCUSSION TITLE
Invalid account creation for Github apps

### DISCUSSION THREAD

**Discussion by benhatsor** [community/community#12538] [Other Feature Feedback, Questions, & Ideas]
**Description**

When trying to create an account to authorize a Github app, Github does not return to the app auth screen after finishing account creation.

**Reproduction Steps**

1. Go to Github app auth screen
https://github.com/login?client_id=7ede3eed3185e59c042d&return_to=%2Flogin%2Foauth%2Fauthorize%3Fclient_id%3D7ede3eed3185e59c042d%26scope%3Drepo%252Cuser%252Cwrite%253Aorg
2. Click "Create an account".
3. Fill out account details and click "Create account".
4. Github does not redirect back to the Github auth screen. Instead, it redirects to the repositories page.

**Logs**

Started new account creation process at:

https://github.com/login?client_id=7ede3eed3185e59c042d&return_to=%2Flogin%2Foauth%2Fauthorize%3Fclient_id%3D7ede3eed3185e59c042d%26scope%3Drepo%252Cuser%252Cwrite%253Aorg

On click of “Create account” button, redirected to:

https://github.com/account_verifications?recommend_plan=true

The `return_to` query is dropped and ignored.
Query is replaced with `recommend_plan=true`.

**Comment by stevepopovich** [comment]
Hi Bar! Is this specific to the GitHub mobile app?

**Reply by benhatsor** [reply]
I’ve checked and I believe it doesn’t work on desktop as well. Changed the category to General
